src/AppBundle/Controller/Security/AdherentSecurityController.php

```diff
@@ -46,4 +55,70 @@ public function loginCheck()
     public function logoutAction()
     {
     }
+
+    /**
+     * @Route("/mot-de-passe-oublie", name="adherent_forgot_password")
+     * @Method("GET|POST")
+     */
+    public function forgotPasswordAction(Request $request)
```

Comment:
Can you name the action with a verb please like `retrieveForgotPasswordAction()`.